src/main/java/org/terasology/structureTemplates/internal/systems/StructureTemplateEditorServerSystem.java

```diff
@@ -341,7 +344,7 @@ public void onCreateStructureSpawnItemRequest(CreateStructureSpawnItemRequest ev
         EntityBuilder entityBuilder = entityManager.newBuilder("StructureTemplates:structureSpawnItem");
         addComponentsToTemplate(structureTemplateOriginEntity, structureTemplateOriginComponent, blockComponent, entityBuilder);
         // TODO make it optional
-        entityBuilder.addOrSaveComponent(new FallingBlocksPlacementAlgorithmComponent());
+        //entityBuilder.addOrSaveComponent(new NoConstructionAnimationComponent());
```

Comment:
Best to not leave commented out code in place unless it is for some clear purpose (like another comment explaining why it is commented out - temporarily or not)